### PR TITLE
NEP-159 open respective news when the notification is clicked

### PR DESCRIPTION
**Pull request by dev-siris**
@syuraj  @shrestha2lt8  sir few files are added like fcmService,NavigationService and added some code in time.js inorder to work.

### CODE REVIEW

**Comment by syuraj** [comment]
@dev-siris can you address `deepscan` issues?

File: src/services/bgMessaging.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import firebase from 'react-native-firebase';
+// Optional flow type
+
+export default async (message) => {
+    // handle your message
+
+    return Promise.resolve();
+}
```
Comment (syuraj) [inline src/services/bgMessaging.js]: is the file necessary?

Comment (dev-siris) [reply src/services/bgMessaging.js]: not may be in the future for handling background notification i would be useful.. should i delete it or leave it.. ?

Comment (syuraj) [reply src/services/bgMessaging.js]: delete

**Comment by syuraj** [comment]
fyi, I haven't received any notification in beta app since 3 days

File: src/services/bgMessaging.js (hunk 1 above)
Comment (dev-siris) [reply src/services/bgMessaging.js]: okay

Comment (dev-siris) [reply src/services/bgMessaging.js]: aru kei garna parxa sir.. article ko duplicate key ko issue ayee rako xa.. tesai lay notification na aako hola..

Comment (syuraj) [reply src/services/bgMessaging.js]: article ko insert garda duplicate key issue? tyo paila dekhi thiyo, kasari yo sanga related cha?
Notification ma data (article) thulo bhahekole ho. Timro local testing ma kaam garyo?

Comment (dev-siris) [reply src/services/bgMessaging.js]: mero local ma kaam gari rako xa.. so mailay pr create gareko ho..

Comment (syuraj) [reply src/services/bgMessaging.js]: putting whole article into data of notification only works sometimes
can you try putting large data in notification and see if it works?
I think we might need to make another graphql api call from article detail
or store the articles in mobile cache.

Comment (syuraj) [reply src/services/bgMessaging.js]: @chetannn do you have android phone with beta app installed https://play.google.com/store/apps/details?id=com.siristechnology.nepaltodayapp.beta?
if so, are you getting notifications from beta app? 
if not, can you install it? https://play.google.com/apps/internaltest/4701173727091738619

Comment (dev-siris) [reply src/services/bgMessaging.js]: yes.. sir i have recently wrote an integration test which leads to the problem of large content in data of the notification.

Comment (dev-siris) [reply src/services/bgMessaging.js]: sir.. data ma chai id matra pathayera paxi.. notification open garda chai query garna parla

**Pull request closed without merging**